mlflow/protos/unity_catalog_oss_messages.proto

```diff
@@ -0,0 +1,92 @@
+syntax = "proto2";
+
+package mlflow.unitycatalog;
+
+import "scalapb/scalapb.proto";
+import "databricks.proto";
+
+option java_package = "com.databricks.api.proto.managedcatalog";
+option java_generate_equals_and_hash = true;
+option (scalapb.options) = {
+  flat_package: true,
+};
+
+// Registered Model Management API
+
+message CreateRegisteredModel {
+  optional RegisteredModelInfo registered_model_info = 1;
+
+  message Response {
+    optional RegisteredModelInfo registered_model_info = 1;
+  }
+}
+
+message RegisteredModelInfo {
+  // Input fields:
+
+  // [Create:REQ Update:OPT] Name of Model, relative to parent Schema.
+  optional string name = 1;
+  // [Create:REQ Update:IGN] Name of parent Catalog.
+  optional string catalog_name = 2;
+  // [Create:REQ Update:IGN] Name of parent Schema relative to its parent Catalog.
+  optional string schema_name = 3;
+  // [Create:IGN Update:OPT] Username of current owner of Model.
+  optional string owner = 4;
+  // [Create,Update:OPT] User-provided free-form text description.
+  optional string comment = 5;
+  // [Create:OPT] Registered model tags
+  repeated TagKeyValue tags = 20;
+
+  // Output-only fields:
+
+  // [Create,Update:IGN] Storage root URL for model
+  optional string storage_location = 6;
+  // [Create,Update:IGN] Unique identifier of parent Metastore.
+  optional string metastore_id = 7;
```

Comment:
We'll need to keep the OSS version of these messages in sync with this: https://github.com/unitycatalog/unitycatalog/pull/270/files#diff-8d03742f9785228675d14f784a7c0affcc9400e8571603bde6ede194c052d985R1841

Reply:
I haven't landed that PR yet, but soon it will be.